tests/lifecycleHooks.lua

```diff
@@ -0,0 +1,241 @@
+local TestEZ = require(script.Parent.Parent.TestEZ)
+
+local function expectShallowEquals(array1, array2)
+	local function shallowEquals()
+		-- shallow equals between lifecycleOrder and passed array
```

Comment:
This comment is pretty specific for what looks like a generic utility!

Reply:
another reason why comments and refactoring dont do well together